packages/email-service/jest.config.ts

```diff
@@ -12,4 +12,7 @@ module.exports = {
   },
   moduleFileExtensions: ['ts', 'js', 'html'],
   coverageDirectory: '../../coverage/packages/email-service',
+  collectCoverage: true,
+  collectCoverageFrom: ['src/**/*.{js,jsx,ts,tsx}'],
+  coverageReporters: ['text'],
```

Comment:
Why not also `html`? Just asking.

Reply:
it can be added, but `text` is responsible for printing report to the console